### ISSUE TITLE
Invalid paths for set-up (windows)

### ISSUE THREAD

**Issue by Grobson97** [Anirbancosmo/Limpy#1]
Hi,

I'm on a windows operating system and I tried cloning your repository from github as detailed in the README. 

After the clone command there is an error saying "error: invalid path 'limpy/.../halos_L205_z7.00.npz'". This for some reason causes all the folders to be deleted after the clone. This can be solved by then restoring (see attached image). However, this still has an issue with the folder named "..." inside limpy.

I see there is a folder named "data" which has the same data files as the "..." folder. Is there any difference to these? Windows cannot have folders called "...".

Also, when running an example notebook I have issues with the import limpy.[submodule] lines as i believe something is pointing to these folders within the invalid folder name, thus preventing imports.

Is it possible to remove the "..." folder or rename it?

Thanks,
Gethin Robson.

![limpy_install_terminal_errors](https://github.com/Anirbancosmo/Limpy/assets/45102919/6a594c27-7846-441f-bc9e-ff8f7ee0c52f)

**Comment by Anirbancosmo** [comment]
Hi,
I have made some modifications to the setup.py file in order to correctly include the data files. I have tested it on Unix OS and it appears to be functioning properly. Please clone the latest version of Limpy and reinstall it. I hope that it will work on Windows OS as well.

**Comment by Grobson97** [comment]
Hi, 

Thanks, it's sort of working now. However the data folder within limpy does not have the halos data file so many of the examples still don't work. But the imports are working now.

The setup process I found worked for me is as follows:
git clone https://github.com/Anirbancosmo/Limpy.git
cd Limpy
git restore --source=HEAD :/
python -m venv venv
.\venv\Scripts\activate
pip install camb colossus astropy matplotlib ipykernel
python setup.py install

It still has the issue with an invalid path preventing checkout due to the folder within limpy called "..." but those aren't important for the examples now it seems. This is why I've had to use the git restore line.

It does look like the halo file isn't in the limpy\data folder, and this stops the examples working.

Thanks,
Gethin.
